### PR TITLE
Normalize Challenges for COM1000

### PR DESCRIPTION
**Pull request by SaintPeter**
This does two things:
1) Normalizes the property order for all challenge files
2) Removes non-required, blank properties from all challenge files

Tested locally to the best of my ability (repeatedly).

Let's get this thing accepted so we can move on with our lives.

### CODE REVIEW

File: seed/challenges/02-data-visualization-certification/react.json [hunk 1]
```diff
@@ -1,30 +1,19 @@
 {
   "name": "React",
   "order": 14,
-  "isComingSoon": true,
+  "isComingSoon": "true",
```
Comment (bugron) [inline seed/challenges/02-data-visualization-certification/react.json]: `isComingSoon` is defined as `boolean`, https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/common/models/challenge.json#L36-L38. Not sure if this makes sense but I think it's better remove double quotes. The same thing everywhere.

File: seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json [hunk 2]
```diff
@@ -5,46 +5,37 @@
...
       "descriptionEs": [
         "Tanto Chrome como Firefox tienen excelentes consolas JavaScript, también conocidas como DevTools, para depurar tu código JavaScript.",
         "Puedes encontrar las Herramientas para desarrolladores (<code>Developer tools</code>) en el menú de Chrome o la Consola web (<code>Web Console</code>) en el menú de FireFox. Si estás utilizando un navegador diferente, o un dispositivo móvil, nuestra recomendación es que cambies a la versión de escritorio de Firefox o Chrome.",
         "Vamos a imprimir en esta consola utilizando el método <code>console.log</code>.",
         "<code>console.log('Hello world!')</code>"
-        ],
-      "namePt": "",
-      "descriptionPt": []
+      ],
+      "difficulty": "0"
```
Comment (bugron) [inline seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json]: This must be removed.

File: seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json [hunk 3]
```diff
@@ -81,9 +66,8 @@
         "<code>console.log(typeof(0));</code>",
         "<code>console.log(typeof([]));</code>",
         "<code>console.log(typeof({}));</code>"
-        ],
-      "namePt": "",
-      "descriptionPt": []
+      ],
+      "difficulty": "0"
```
Comment (bugron) [inline seed/challenges/03-back-end-development-certification/automated-testing-and-debugging.json]: The same thing here. This must be removed.

File: seed/challenges/hikes/computer-basics.json [hunk 4]
```diff
@@ -130,30 +129,30 @@
...
-      "nameFr": "",
-      "descriptionFr": [],
-      "nameRu": "",
-      "descriptionRu": [],
-      "nameEs": "",
-      "descriptionEs": [],
-      "namePt": "",
-      "descriptionPt": []
+        [
+          "1 === on and 0 ===off",
```
Comment (bugron) [inline seed/challenges/hikes/computer-basics.json]: `0 ===off` -> `0 === off`

Comment (SaintPeter) [inline seed/challenges/hikes/computer-basics.json]: I'm not going to edit existing text.

**Comment by SaintPeter** [comment]
@benmcmahon100 Can we look at booleans?

**Comment by SaintPeter** [comment]
Removed the two "Difficulty" properties.

File: seed/challenges/hikes/computer-basics.json (hunk 4 above)
Comment (bugron) [inline seed/challenges/hikes/computer-basics.json]: OK

**Comment by SaintPeter** [comment]
Fixed the Booleans.

**Comment by ltegman** [comment]
I'm still seeing the booleans as strings in this PR.

**Comment by SaintPeter** [comment]
I think those are on the top level - Ben will fix those as well.  There are also some "non-boolean" booleans inside the hikes.  Those are supposed to be strings.

**Comment by SaintPeter** [comment]
For the love of all that is holy, I believe this crazy thing is correct and complete.

**Comment by bugron** [comment]
:+1:

**Pull request merged**